docs/en/ingest-management/commands.asciidoc

```diff
@@ -72,19 +173,48 @@ This flag is helpful when using automation software or scripted deployments.
 Show help for the `enroll` command.
 
 `--insecure`::
-Allow an insecure connection to {kib}. When this flag is specified, API keys are
-sent in clear text. We strongly recommend that you use a secure connection.
+Allow insecure connections to {fleet-server}. When this flag is specified, API
+keys are sent in clear text. We strongly recommend that you use a secure
+connection.
+
+//QUESTION: Is this true? Are the keys really sent in clear text, or does this
+//have to do with accepting auto-generated certificates?
+
+`--kibana-url <string>`::
+deprecated:[7.13.0] This deprecated setting has been replaced by `--url`.
+
+//QUESTION: In the past, we've left out --staging to avoid confusion. Should I
+//add it here?
+
+`--url <string>`::
+Fleet Server URL to use to enroll the {agent} into {fleet}.
+
+// end::enroll-install-options[]
 
 {global-flags-link}
 
 [discrete]
-=== Example
+=== Examples
+
+Enroll the {agent} in {fleet}:
+
+[source,shell]
+----
+elastic-agent enroll -f \
+  --url=https://cedd4e0e21e240b4s2bbbebdf1d6d52f.fleet.eu-west-1.aws.cld.elstc.co:443 \
+  --enrollment-token=NEFmVllaa0JLRXhKebVKVTR5TTI6N2JaVlJpSGpScmV0ZUVnZVlRUExFQQ==
+----
+
+Enroll the {agent} in {fleet} and set up {fleet-server}:
 
 [source,shell]
 ----
-elastic-agent enroll http://localhost:5601 ZnmNIdzVITUJua2QIdU5FTWROVjY6dHY2N1EybWNTMUdPejg5ODbYcVpNUQ==
+elastic-agent enroll -f --fleet-server-es=http://myhost:9200 \
+  --fleet-server-service-token=AbEAAdesYXN1abMvZmxlZXQtc2VldmVyL3Rva2VuLTE2MTkxMzg3MzIzMTg7dzEta0JDTmZUcGlDTjlwRmNVTjNVQQ
 ----
 
+//TODO: Add examples that show how to use custom certificates.
+
```

Comment:
I can give some examples, this all assumes that they generated the certificates with the following names:

```
ca.crt - Root CA Certificate
fleet-server.crt - Fleet Server Certificate
fleet-server.key - Fleet Server Private Key
```

Start Elastic Agent with Fleet Server (running on a custom CA):

`elastic-agent enroll -f --fleet-server-es=https://elasticsearch:9200 --fleet-server-es-ca=ca.crt --fleet-server-service-token=AbEAAdesYXN1abMvZmxlZXQtc2VldmVyL3Rva2VuLTE2MTkxMzg3MzIzMTg7dzEta0JDTmZUcGlDTjlwRmNVTjNVQQ --fleet-server-cert fleet-server.crt --fleet-server-cert-key fleet-server.key
`

Enroll another Elastic Agent into that bootstrapped Elastic Agent with Fleet Server:

`elastic-agent enroll -f --url https://other-host:8220 --enrollment-token NEFmVllaa0JLRXhKebVKVTR5TTI6N2JaVlJpSGpScmV0ZUVnZVlRUExFQQ== --certificate-authorities root.ca`